com.unity.cinemachine/Documentation~/CinemachineInputAxisController.md

```diff
@@ -28,3 +29,122 @@ If you like, you can also use CinemachineInputAxisController with your own scrip
+    {
+        if (m_Slider != null)
+            return m_Slider.value; // The value returned will change the axis value.
+        
+        return 0;
+    }
+}
+
+public class SliderInputController : InputAxisControllerBase<SliderReader> {} //The component of your reader. Input InputAxisControllerBase handles the setup.
+
+#if UNITY_EDITOR
+[UnityEditor.CustomEditor(typeof(SliderInputController))] // Optional but recommended if want a nice inspector.
+public class SliderControllerEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+## CinemachineCamera and Input system components
+
+We assume you already know how to setup `PlayerInput` and `PlayerInputManager`. All the documentation can be found on the Input System package documentation page and samples.
+
+### Read from PlayerInput component
+
+You will need a custom InputAxisController that receives axis from the PlayerInput component. Add this script to your CinemachineCamera and assign the `PlayerInput` field. The `behaviour` field must be set to `InvokeCSharpEvents`.
+
+```cs
+using UnityEngine;
+using Unity.Cinemachine;
+using System;
+using UnityEngine.InputSystem;
+
+class PlayerInputReceiver : InputAxisControllerBase<SimpleReader>
+{
+[SerializeField]
+private PlayerInput m_PlayerInput;
+
+    private void Awake()
+    {
+        m_PlayerInput.onActionTriggered += OnActionTriggered; // Call back when the PlayerInput receives an Input.
+    }
+
+    public void OnActionTriggered(InputAction.CallbackContext value)
+    {
+        foreach (var controller in Controllers)
+        {
+            controller.Input.SetValue(value.action); // Send the Input to all the controllers.
+        }
+    }
+}
+
+[Serializable]
+class SimpleReader : IInputAxisReader
+{
+    private Vector2 m_Value; // Assumes the action is a Vector2 for simplicity but can be changed for a float.
+    [SerializeField]
+    private InputActionReference m_Input;
+    
+    public void SetValue(InputAction action)
+    {
+        if (m_Input!= null && m_Input.action.id == action.id)
+            m_Value = action.ReadValue<Vector2>(); // Is the input referenced in the inspector matches the updated one update the value.
+    }
+
+    public float GetValue(UnityEngine.Object context, int playerIndex, bool autoEnableInput, IInputAxisOwner.AxisDescriptor.Hints hint)
+    {
+        if(hint == IInputAxisOwner.AxisDescriptor.Hints.X)
+            return m_Value.x * 100;
+        
+        return m_Value.y * 100;
+    }
+}
+
+#if UNITY_EDITOR
+[UnityEditor.CustomEditor(typeof(PlayerInputReceiver))]
+public class PlayerInputReceiverEditor : Unity.Cinemachine.Editor.InputAxisControllerEditor {}
+#endif
+```
+
+### Local multiplayer
+
+To use the previous script in a multiplayer environment you will need one `CinemachineBrain` attached on the Player prefab Camera and increase the channel mask of both the `CinemachineCamera` and the `CinemachineBrain`.
```

Comment:
Can you elaborate on this line?  I think it needs to be spelled out a little more.  Also, what does one do with the following script?

Reply:
This doc assumes you are familiar with this setup. In an ideal world I would just reference this page. I am wandering what happens if it get's updated? Maybe you have an idea? 
https://docs.unity3d.com/Packages/com.unity.inputsystem@1.5/manual/PlayerInput.html